### ISSUE TITLE
Except and Intersect fails when any of argument sequences contain null elements

### ISSUE THREAD

**Issue by atifaziz** [atifaziz/LINQBridge#24]
```
The following test will fail:

            var first = Read("abc", null);
            var second = Read("bcd", null);
            first.Except(second).AssertEquals("abc");

The reason is the same as in the issue #21: Intersect and Except operations 
rely on generic Dictionary as a substitute for the Set, but the Dictionary is 
not suitable for null key values.
```

Original issue reported on code.google.com by `mrdont@mail.ru` on 27 Jun 2011 at 6:15

**Comment by atifaziz** [comment]
Original comment by `azizatif` on 16 Apr 2012 at 6:05
- Changed state: **Accepted**
- Added labels: **Milestone-Release1.3**

**Comment by atifaziz** [comment]
```
This issue was closed by revision f8234998bc9c.
```

Original comment by `azizatif` on 16 Apr 2012 at 7:12
- Changed state: **Fixed**
